optashift-employee-rostering-webapp/pom.xml

```diff
@@ -46,6 +46,50 @@
     </dependency>
   </dependencies>
 
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-war-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>war-exploded</id>
+            <phase>prepare-package</phase>
+            <goals>
+              <goal>exploded</goal>
+            </goals>
+            <configuration>
+              <webappDirectory>${wildfly.home}/standalone/deployments/${project.build.finalName}.war</webappDirectory>
```

Comment:
By default, the exploded war directory is generated by the maven-war-plugin, IIRC.
So instead of needing this configuration by exception, can we avoid configuring the war-plugin by telling the wildfly plugin to take the standard exploded war dir? (And why isn't it doing that by default already?)

Less is more.